build/configs/imxrt1020-evk/bootstrap.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -e
+
+
+CURDIR=$(readlink -f "$0")
+CURDIR_PATH=$(dirname "$CURDIR")
+
+BINFILE=${CURDIR_PATH}/ivt_flashloader.bin
+SDPHOST=${CURDIR_PATH}/sdphost
+BLHOST=${CURDIR_PATH}/blhost
+BINFILE=${CURDIR_PATH}/ivt_flashloader.bin
+
+BASEADDR=0x20208000
+IVTADDR=0x20208400
+TTYDEV="/dev/ttyACM0"
+SUDO=sudo
+
+echo "Connecting the chip ROM ... "
```

Comment:
Have some check for connected board and build config used, currently if build for 1050 config and try to flash on 1020 there is no proper error status.

Reply:
fixed

Reply:
fixed

Reply:
I'm not getting any error message, can you share the failure logs here?

Reply:
########################################
 WARNING:If download fails, don't forget to replug
 and reset the board before next try!!
 #################################################

Using partitions from .config file

Using sizes from .config file

offsets: 0x60000000 0x60040000 0x60080000 0x600C0000 0x60100000 0x60140000
sizes: 256 256 256 256 256 256
partitions: kernel app micom micom wifi wifi
Connecting the chip ROM ...
Status (HAB mode) = 1450735702 (0x56787856) HAB disabled.
Reponse Status = 4042322160 (0xf0f0f0f0) HAB Success.

Download /home/siju.cherian/Tizen/work/TizenRT/TizenRT/build/configs/imxrt1050-evk/../../tools/nxp/ivt_flashloader.bin to 0x20000000
Preparing to send 90039 (0x15fb7) bytes to the target.
(1/1)1%unexpected write file response 0x89232389.
Status (HAB mode) = 1450735702 (0x56787856) HAB disabled.
Reponse Status = 2300781449 (0x89232389) REENUM Ack

Start at 0x20000400
Status (HAB mode) = 1450735702 (0x56787856) HAB disabled.

wait ...
Error: Initial ping failure: No response received for ping command.Makefile.unix:533: recipe for target 'download' failed
make: *** [download] Error 1

Reply:
bsvt18@ubuntu:~/TizenRT/PublicGit/June6/os$ make download

########################################
 WARNING:Make sure NO other application like 
 putty etc, is accessing ttyACM0. If download 
 fails, reset the board before next try!!
 #################################################

Target is NOT IMXRT1020_EVK
Makefile.unix:533: recipe for target 'download' failed
make: *** [download] Error 1

Reply:
The actual code that validates this is 
if [[ "${CONFIG_ARCH_BOARD_IMXRT1020_EVK}" != "y" ]]
FIND_WORD="RT1020-EVK"	then
	echo "Target is NOT IMXRT1020_EVK"
	exit 1
fi